### DISCUSSION TITLE
Support for Inverter: MICROTEK GTI 3.3KW-M11

### DISCUSSION THREAD

**Discussion by PV-Joe** [davidrapan/ha-solarman#58] [Ideas]
I just found the support for MICROTEK GTI 3.3KW-M11 Inverter here:
[https://github.com/StephanJoubert/home_assistant_solarman/issues/639](uel)

The guy confirmed to use “sofar_lsw3.yaml”

Maybe you would like to add it as a new yaml to extend the list of supported inverters?

**Comment by davidrapan** [comment]
No kidding cause those inverters are under the hood all the fking same, lol.

I'm somehow convinced that even register numbers are artificially changed by brands, lol. 😆

Edit: I placed it in between other Supported Inverters in the wiki. Thanks! 😉

**Reply by PV-Joe** [reply]
Yes, many have the same hardware and are just labeled differently.
That makes it easier for your list of supported devices.
Just copy and rename the YAML and that’s it.
Easy way to collect more users of your fantastic integration 😄

**Reply by davidrapan** [reply]
Thanks, it's really nice of you to say that. 🥂
